services/analyticsproviders/posthog/src/main/kotlin/io/element/android/services/analyticsproviders/posthog/PosthogAnalyticsProvider.kt

```diff
@@ -66,10 +66,10 @@ class PosthogAnalyticsProvider @Inject constructor(
     }
 
     override fun updateUserProperties(userProperties: UserProperties) {
-        posthog?.identify(
-            REUSE_EXISTING_ID, userProperties.getProperties()?.toPostHogUserProperties(),
-            IGNORED_OPTIONS
-        )
+//        posthog?.identify(
+//            REUSE_EXISTING_ID, userProperties.getProperties()?.toPostHogUserProperties(),
+//            IGNORED_OPTIONS
+//        )
```

Comment:
This was intended, right?

Reply:
Yes for the moment analytics id is anonymous.